### DISCUSSION TITLE
Recipe by users and allow searches by users?

### DISCUSSION THREAD

**Discussion by Jucgshu** [mealie-recipes/mealie#897] [Q&A] [answered]
Hi guys,

Awfully sorry if it has been discussed somewhere. I've searched the repository without any luck so far. Is there any way to:
1/ filter searches by users?
2/ display the usernames inside the recipe?

Thanks!

**Comment by hay-kot** [comment, marked as answer]
In the most recent release recipes aren't attached to users. This will be changing in v1 and this functionality will come at some-point. If someone want's to work on it in the `mealie-next` branch feel free.

**Comment by Jucgshu** [comment]
Thanks for your answer @hay-kot . Kudos for your work!
